eng/scripts/automation/GenerateAndBuildLib.ps1

```diff
@@ -1,5 +1,9 @@
 #Requires -Version 7.0
 $CI_YAML_FILE = "ci.yml"
+
+#mgmt: resourceProvider to sdk package name map
+$packageNameHash = [ordered]@{"vmware" = "avs"; "azure-kusto"="kusto"; "cosmos-db"="cosmosdb";"customer-insights"="customerinsights";"monitor"="insights";"msi"="managedserviceidentity";"web"="appservice"}
```

Comment:
I recommend putting each element on separate lines (semicolon is optional then). It makes it much easier to read, and future diffs to be smaller. Of course, I agree with @weshaggard as well: this should come from configuration.